crates/javy/src/lib.rs

```diff
@@ -40,6 +40,7 @@ pub use config::Config;
 pub use quickjs_wasm_rs as quickjs;
 pub use runtime::Runtime;
 
+pub mod alloc;
```

Comment:
Why don't we instead add `[cfg(feature = "export_alloc_fns")]` here and remove the cfg directive from each of the function definitions?

Reply:
One small reason is if they don't enable the `export_alloc_fns` feature, they can still add exported functions in their own crate to call `canonical_abi_realloc` and `canonical_abi_free` but export them with a name of their own choosing instead of with the export name we chose.